_partials/_create-hypertable-blockchain.md

```diff
@@ -36,7 +36,7 @@ with Timescale tables similar to standard PostgreSQL.
     the timestamp data to use for partitioning:
 
     ```sql
-    SELECT create_hypertable('transactions', 'time');
+    SELECT create_hypertable('transactions', by_range('time'));
```

Comment:
Do we want this to be the premier way of doing things going forward? Even when there's a clear time-oriented use case? 

Shouldn't we explain somewhere that, e.g., `SELECT create_hypertable('transactions', 'time');` is a shortcut for `SELECT create_hypertable('transactions', by_range('time'));` and that `by_range` is the default for the first dimension?

Reply:
> Do we want this to be the premier way of doing things going forward? Even when there's a clear time-oriented use case?

The new interface is more robust to changes going forward and allow overloading and specialization in various ways. It also support all the existing use-cases, so I see no reason to not use the new interface also for time-oriented use-cases.

> Shouldn't we explain somewhere that, e.g., `SELECT create_hypertable('transactions', 'time');` is a shortcut for `SELECT create_hypertable('transactions', by_range('time'));` and that `by_range` is the default for the first dimension?

I have no strong opinion on how to formulate the distinction or what overloads to keep and am open to suggestions. The only matter that I think is important is that we need to do an overhaul of the documentation to make it more useful and approachable for wider use-cases.

Reply:
I am worried that we go all in on rewriting the documentation for the new API, users will be confused and they don't understand how the existing API relates to the new one. 

I think, for most cases, where you only have a time column, it seems more simple and convenient to use the existing API. So, it seems reasonable to think of it as an abbreviated version of the more general API rather than saying it is legacy.

Reply:
>I think, for most cases, where you only have a time column, it seems more simple and convenient to use the existing API. So, it seems reasonable to think of it as an abbreviated version of the more general API rather than saying it is legacy.

I'm fine with this. It is going to be the first step anyway, so we can use this for now. Can you point to the places where you think we should update the PR in that case?